mozdownload/scraper.py

```diff
@@ -944,23 +1012,6 @@ def detect_platform(self):
 
         return platform
 
-    def get_build_info_for_index(self, build_index=None):
-        """Get additional information for the build at the given index."""
-        url = urljoin(self.base_url, self.build_list_regex)
-
-        self.logger.info('Retrieving list of builds from %s' % url)
-        parser = self._create_directory_parser(url)
-        parser.entries = parser.filter('.*-%s$' % self.changeset)
-
-        if not parser.entries:
-            raise errors.NotFoundError('No builds have been found', url)
-
-        self.show_matching_builds(parser.entries)
-
-        self.logger.info('Selected build: %s' % parser.entries[0])
-
-        return (parser.entries, 0)
-
```

Comment:
Via Treeherder we have about 4 months of history for getting try builds. I don't think scraping the server makes sense anymore given that this takes ages to actually get the builds.